_posts/2018-04-10-building-the-peer-to-peer-internet.md

```diff
@@ -0,0 +1,28 @@
+---
+layout: post
+title: Building The Peer-to-Peer Internet
+slug: building-the-peer-to-peer-internet
```

Comment:
shorten to `peer-to-peer-internet`?

Reply:
We try to keep the slug exactly the same as title so far?

Reply:
We shorten slugs all the time